src/modules/next-template/components/Button.tsx

```diff
@@ -0,0 +1,21 @@
+import styled from 'styled-components';
+
+export default function Button({ content }: any) {
```

Comment:
This most likely will be reused everywhere, you can group this as `src/common/components/buttom/Button.tsx`